torch/csrc/jit/script/sugared_value.cpp

```diff
@@ -509,6 +511,31 @@ std::shared_ptr<SugaredValue> NamedTupleConstructor::call(
   return std::make_shared<SimpleValue>(self);
 }
 
+std::shared_ptr<BuiltinFunction> BuiltinFunction::tryCreate(
+    Symbol symbol,
+    c10::optional<NamedValue> self) {
```

Comment:
Will `self` ever be `c10::nullopt` here ?

Reply:
Not in the current PR, but it seems like a good idea to have the arguments to tryCreate match the arguments to BuiltinFunction::create so that the description of what it does is clear.